CHANGES.txt

```diff
@@ -27,6 +27,11 @@ Breaking Changes
 Changes
 =======
 
+ - Added Privileges plugin for the Enterprise Edition.
```

Comment:
pls note that this is admin-ui related, also a user don't know about a 'plugin', I think its a main menu entry/section.